index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="fr">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ohmyfood</title>
+    <meta
+      name="description"
+      content="Ohmyfood, entreprise de commande de repas en ligne"
+    />
+    <link rel="stylesheet" href="./.public/css/style.css" />
+    <script
+      src="https://kit.fontawesome.com/b2f281879d.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <!--      HEADER     -->
+
+    <header>
+      <img src="images/logo/ohmyfood.png" alt="logo du site" />
+    </header>
+
+    <!--     MAIN     -->
+
+    <main>
+      <!--     FIND US     -->
+
+      <section id="find-us">
+        <p><i class="fas fa-map-marker-alt center"></i>Paris, Belleville</p>
+      </section>
+
+      <!--     ABOUT US     -->
+
+      <section id="about-us">
+        <h1>Réservez le menu qui vous convient</h1>
+        <p>Découvrez des restaurants d'exception, sélectionnés par nos soins</p>
+        <a href="#restaurants">Explorer nos restaurants</a>
+      </section>
+
+      <!--     HOW TO     -->
+
+      <section id="how-to">
+        <h2>Fonctionnement</h2>
+        <ol>
+          <li><i class="fas fa-mobile-alt"></i>Choisissez un restaurant</li>
+          <li><i class="fas fa-list-ul"></i>Composez votre menu</li>
+          <li><i class="fas fa-store"></i>Dégustez au restaurant</li>
+        </ol>
+      </section>
+
+      <!--     RESTAURANTS     -->
+
+      <section id="restaurants">
+        <h2>Restaurants</h2>
+
+        <!--      PALETTE DU GOUT     -->
+
+        <article>
+          <figure>
+            <a href="./pages/la-palette-du-gout.html">
+              <span>Nouveau</span>
+              <img
+                src="images/restaurants/jay-wennington-N_Y88TWmGwA-unsplash.jpg"
+                alt="plat fin gastronomique et deux verres de vin"
+              />
+            </a>
+            <figcaption>
+              <a href="./pages/la-palette-du-gout.html">
+                <h3>La palette du goût</h3>
+                <p>Ménilmontant</p>
+              </a>
+              <i class="far fa-heart"></i>
```

Comment:
You have to add also heart plain and put the two i inside a span

Reply:
ok got it